### ISSUE TITLE
Support bazel 2.0

### ISSUE THREAD

**Issue by stephendl** [salesforce/bazel-springboot-rule#15]
Firstly, thank you for putting together this awesome tool!

My issue: 
As a bazel user
I would like to be able to build a spring boot project with bazel 2.0 and bazel-spring-boot without errors
So that I can use the default google cloud build bazel task for building spring boot applications
And so that I don't have to use an out of date, regressed 0.29.1 bazel task 

**To Reproduce:**

On MacOS 10.15.2, using bazel 2.0 (I believe the same error is given on any bazel version after 0.29.1)

when building the samples/helloworld spring boot application

```
bazel build //samples/helloworld

Starting local Bazel server and connecting to it...
ERROR: /Users/me/src/bazel-springboot-rule/WORKSPACE:9:1: name 'maven_server' is not defined
ERROR: error loading package '': Encountered error while reading extension file 'tools/build_defs/repo/http.bzl': no such package '@bazel_tools//tools/build_defs/repo': error loading package 'external': Could not load //external package
ERROR: error loading package '': Encountered error while reading extension file 'tools/build_defs/repo/http.bzl': no such package '@bazel_tools//tools/build_defs/repo': error loading package 'external': Could not load //external package
INFO: Elapsed time: 2.291s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
```

**Comment by plaird** [comment]
Thanks Stephen for the feedback, we are glad you can make use of the springboot rule!

Bazel 2.0 removed the maven_jar/maven_server native rules, which is causing the error above. Internally at Salesforce, we are using the springboot rule as-is with Bazel 1.0 so you should be able to use that version. We are actively trying to move to the rules_jvm_external/maven_install this sprint in our repo, but have hit a snag so we still aren't there.

Another member of the community reported this and published a fork with the changes necessary for Bazel 2.0. See Issue #13. We hope to merge those changes to master soon. Just need a few spare cycles.

If you want to use 2.0 please try the workaround presented in #13. Or, try 1.0 again and make sure it works. Post back if either solution worked for you. Thanks!

**Comment by stephendl** [comment]
Thank you for your prompt reply!

My mistake... the issues I had with 1.0.0 -> 1.2.1 were https related (since Jan 15th maven repos at apache & maven only support https) 

After fixing that in the WORKSPACE file (line 11) I can get `samples/helloworld` to run using bazel 1.2.1. That'll work for me for now. I'll try the fix in issue [#13](https://github.com/salesforce/bazel-springboot-rule/issues/13) once my build is green again!

Thanks again for the prompt reply, and the great work you guys are doing. I appreciate it.

**Comment by plaird** [comment]
Great news! I will close this issue with the follow-up work (for 2.0) being tracked in #13. Thanks again for filing an issue. If you hit this, it is likely other users will too.
